### DISCUSSION TITLE
Issue with OTP Not Being Received During Sign-In on UpYog Domain

### DISCUSSION THREAD

**Discussion by KeertanaITL** [upyog/UPYOG#1113] [Q&A]
I'm encountering an issue with the OTP (One-Time Password) not being received during the sign-in process on the UpYog domain after completing the Bootstrap setup. Here's what I've done so far:

Completed the Bootstrap setup for UpYog.
Opened the domain to access the sign-in page.
Entered the required details and requested an OTP.
Unfortunately, the OTP is not being received. I've checked my network connectivity and ensured that my phone number/email is correct. I've also looked into my spam/junk folder but to no avail.

Has anyone else experienced this issue? What steps can I take to resolve this and successfully receive the OTP?

**Comment by vinothrallapalli-eGov** [comment]
If you want to receive SMS, then you need to integrate with an SMS provider. In the data bootstrap collection, we have already provided the citizen creation postman collection. Please port forward a user service and run the postman collection then Citizen/Employee user will be created and using same, you can login. After citizen register then you can login using default OTP 123456.

**Comment by KeertanaITL** [comment]
I tried using default otp but it is not working.
________________________________

**Reply by vinothrallapalli-eGov** [reply]
Hi Keerthana,

   For citizen registration the default OTP won't work. Once the citizen registered in the system then only it work. Please use following curl to create a citizen. After below script is executed use mobile no 9999999999 to login.

curl --location 'http://localhost:8094/user/users/_createnovalidate' \
--header 'Content-Type: application/json' \
--data-raw '{
  "requestInfo": {
    "apiId": "Rainmaker",
    "ver": ".01",
    "ts": null,
    "action": "_update",
    "did": "1",
    "key": "",
    "msgId": "20170310130900|en_IN",
    "authToken": "",
            "userInfo": {"id":1792,"uuid":"300a68c4-3517-49f4-8ee1-f00d8168010a","userName":"EMP-1013-000157","name":"Bill Creator ","mobileNumber":"9738010958","emailId":null,"locale":null,"type":"EMPLOYEE","roles":[{"name":"EGF Bill Creator","code":"EGF_BILL_CREATOR","tenantId":"pg.citya"}],"active":true,"tenantId":"pg.citya","permanentCity":null}
  },
  "user": {
    "userName": "9999999999",
    "name": "Vinoth R",
    "gender": "Male",
    "mobileNumber": "9999999999",
    "type": "CITIZEN",
    "active": true,
    "password": "eGov@4321",
    "roles": [
      {
        "name": "Citizen",
        "code": "CITIZEN",
        "tenantId": "pg"
      }
    ],
    "tenantId": "pg"
  }
}'

**Comment by KeertanaITL** [comment]
thank you got it.
________________________________
